### PR TITLE
decrease react and react-dom version dependency

### PR DESCRIPTION
**Pull request by praneybehl**
Hi,
Great work by the way, I am trying to using this with Meteor with is currently at v0.14.1 of React and the build fail.

Would it be ok to decrease the v dependency to v^0.14.1 ?

Cheers!

### CODE REVIEW

**Comment by praneybehl** [comment]
Aborting since it doesn't work with broserify.

**Comment by azazdeaz** [comment]
Hi @praneybehl,

It should work with any React v^0.14.0. What is your problem with it?

Can you help me with some info about why it is not working with Browserify?

File: package.json [hunk 1]
```diff
@@ -61,8 +61,8 @@
     "node-libs-browser": "^0.5.2",
     "radium": "^0.15.1",
     "raw-loader": "^0.5.1",
-    "react": "^0.14.3",
-    "react-dom": "^0.14.3",
+    "react": "^0.14.1",
+    "react-dom": "^0.14.1",
     "react-matterkit": "0.0.34",
```
Comment (azazdeaz) [inline package.json]: These are just dev dependencies. They are shouldn't matter if you just install ```react-gsap-enhancer``` with npm.

**Comment by praneybehl** [comment]
thanks for getting back, I am trying to use it with Meteor + React(from the meteor react package) with browserify project but I get these errors:

```
 While building package npm-container:
   error: couldn't run `npm shrinkwrap`: Command failed: npm ERR! Error: Problems were encountered
   npm ERR! Please correct and try again.
   npm ERR! missing: deep-equal@^1.0.0, required by history@1.13.1
   npm ERR! missing: invariant@^2.0.0, required by history@1.13.1
   npm ERR! missing: warning@^2.0.0, required by history@1.13.1
   npm ERR!     at shrinkwrap_
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/shrinkwrap.js:30:15)
   npm ERR!     at /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/shrinkwrap.js:24:5
   npm ERR!     at /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/ls.js:45:30
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:130:5
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR! If you need help, you may report this *entire* log,
   npm ERR! including the npm and node versions, at:
   npm ERR!     <http://github.com/npm/npm/issues>
   
   npm ERR! System Darwin 15.0.0
   npm ERR! command "/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node"
   "/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/npm" "shrinkwrap"
   npm ERR! cwd /Users/Praney/projects/explainee_reactjs/app/packages/npm-container/.npm/package-new-1nihgz0
   npm ERR! node -v v0.10.40
   npm ERR! npm -v 1.4.28
   npm ERR! 
   npm ERR! Additional logging details can be found in:
   npm ERR!     /Users/Praney/projects/explainee_reactjs/app/packages/npm-container/.npm/package-new-1nihgz0/npm-debug.log
   npm ERR! not ok code 0
   npm ERR! Error: Problems were encountered
   npm ERR! Please correct and try again.
   npm ERR! missing: deep-equal@^1.0.0, required by history@1.13.1
   npm ERR! missing: invariant@^2.0.0, required by history@1.13.1
   npm ERR! missing: warning@^2.0.0, required by history@1.13.1
   npm ERR!     at shrinkwrap_
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/shrinkwrap.js:30:15)
   npm ERR!     at /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/shrinkwrap.js:24:5
   npm ERR!     at /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/ls.js:45:30
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:130:5
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR! If you need help, you may report this *entire* log,
   npm ERR! including the npm and node versions, at:
   npm ERR!     <http://github.com/npm/npm/issues>
   
   npm ERR! System Darwin 15.0.0
   npm ERR! command "/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node"
   "/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/npm" "shrinkwrap"
   npm ERR! cwd /Users/Praney/projects/explainee_reactjs/app/packages/npm-container/.npm/package-new-1nihgz0
   npm ERR! node -v v0.10.40
   npm ERR! npm -v 1.4.28
   npm ERR! 
   npm ERR! Additional logging details can be found in:
   npm ERR!     /Users/Praney/projects/explainee_reactjs/app/packages/npm-container/.npm/package-new-1nihgz0/npm-debug.log
   npm ERR! not ok code 0
   
   
   While building package praneybehl:react-gsap-enhancer:
   error: couldn't run `npm shrinkwrap`: Command failed: npm ERR! Error: Problems were encountered
   npm ERR! Please correct and try again.
   npm ERR! missing: deep-equal@^1.0.0, required by history@1.13.1
   npm ERR! missing: invariant@^2.0.0, required by history@1.13.1
   npm ERR! missing: warning@^2.0.0, required by history@1.13.1
   npm ERR!     at shrinkwrap_
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/shrinkwrap.js:30:15)
   npm ERR!     at /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/shrinkwrap.js:24:5
   npm ERR!     at /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/ls.js:45:30
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:130:5
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR! If you need help, you may report this *entire* log,
   npm ERR! including the npm and node versions, at:
   npm ERR!     <http://github.com/npm/npm/issues>
   
   npm ERR! System Darwin 15.0.0
   npm ERR! command "/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node"
   "/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/npm" "shrinkwrap"
   npm ERR! cwd /Users/Praney/projects/react-packages/react-gsap-enhancer/.npm/package-new-1mwondc
   npm ERR! node -v v0.10.40
   npm ERR! npm -v 1.4.28
   npm ERR! 
   npm ERR! Additional logging details can be found in:
   npm ERR!     /Users/Praney/projects/react-packages/react-gsap-enhancer/.npm/package-new-1mwondc/npm-debug.log
   npm ERR! not ok code 0
   npm ERR! Error: Problems were encountered
   npm ERR! Please correct and try again.
   npm ERR! missing: deep-equal@^1.0.0, required by history@1.13.1
   npm ERR! missing: invariant@^2.0.0, required by history@1.13.1
   npm ERR! missing: warning@^2.0.0, required by history@1.13.1
   npm ERR!     at shrinkwrap_
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/shrinkwrap.js:30:15)
   npm ERR!     at /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/shrinkwrap.js:24:5
   npm ERR!     at /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/lib/ls.js:45:30
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:130:5
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR!     at
   /Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
   npm ERR!     at cb
   (/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
   npm ERR! If you need help, you may report this *entire* log,
   npm ERR! including the npm and node versions, at:
   npm ERR!     <http://github.com/npm/npm/issues>
   
   npm ERR! System Darwin 15.0.0
   npm ERR! command "/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node"
   "/Users/Praney/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/npm" "shrinkwrap"
   npm ERR! cwd /Users/Praney/projects/react-packages/react-gsap-enhancer/.npm/package-new-1mwondc
   npm ERR! node -v v0.10.40
   npm ERR! npm -v 1.4.28
   npm ERR! 
   npm ERR! Additional logging details can be found in:
   npm ERR!     /Users/Praney/projects/react-packages/react-gsap-enhancer/.npm/package-new-1mwondc/npm-debug.log
   npm ERR! not ok code 0

```

**Comment by azazdeaz** [comment]
Sorry, i'm not sure what happening here. Can you give me some more info about how do you get this log?

**Pull request closed without merging**